spaces/S000150/README.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: S000150
+name: Right "closed-ray" topology on $[0,1]\cap\mathbb Q$
+---
+
+The set $X=[0,1]\cap\mathbb Q$ with its order induced from $\mathbb R$, and the topology generated by a base of open sets consisting of the intervals $[x,\rightarrow)$ for $x\in X$.
+
+The open sets are exactly the [upward-closed sets](https://en.wikipedia.org/wiki/Upper_set) in the ordered set $(X,\le)$.  That includes "closed rays" $[x,\rightarrow)$ and "open rays" $(x,\rightarrow)$ for $x\in X$, as well as sets of the form $(\theta,1]\cap\mathbb Q$ for an irrational $\theta\in(0,1)$.
```

Comment:
Should we add the parallel section to 151?

Reply:
Yeah, I thought about it.  It's kind of obvious that sets of the form $(x,\rightarrow)$ and  $(\theta,,1]\cap X$ are open.
But maybe we could mention:
"Note that "closed rays" of the form $[x,\rightarrow)$ for $x\in X\setminus\\{0\\}$ are not open sets."

Thoughts?